wcfsetup/install/files/lib/util/HeaderUtil.class.php

```diff
@@ -163,10 +168,14 @@ public static function parseOutput($output)
         // move script tags to the bottom of the page
         $javascript = [];
         self::$output = \preg_replace_callback(
-            '~<script data-relocate="true"(?P<script>.*?</script>)\s*~s',
+            '~<script data-relocate="true"(?<attributes>[^>]*)>(?P<script>.*?)</script>\s*~s',
```

Comment:
```suggestion
            '~<script data-relocate="true"(?<attributes>[^>]*+)>(?P<script>.*?)</script>\s*~s',
```

Since we’re using a character class, we can take advantage of a possessive quantifier to avoid backtracking.